### PR TITLE
Add examples file for SPNEGO feature doc

### PR DESCRIPTION
**Pull request by Charlotte-Holt**

### CODE REVIEW

File: ref/feature/spnego/examples.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+== Examples
+
+Open Liberty supports many common SPNEGO scenarios and configurations.
+
+=== Limit SPNEGO authentication 
+You can filter HTTP requests to require SPNEGO authentication for only certain requests, web applications, hosts, or user agents.
+
+The following example configures the following things:
```
Comment (ManasiGandhi) [inline ref/feature/spnego/examples.adoc]: 'The following example configures the following things:'
Maybe rephrase this sentence. How about 'the example provided configures the following things/points'?

File: ref/feature/spnego/examples.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<1> SPNEGO authentication is used for any requests that are received containing the `example.com` hostname for resources within the `protectedApp` web application.
```
Comment (ManasiGandhi) [inline ref/feature/spnego/examples.adoc]: "SPNEGO authentication is used for any requests that are received containing the `example.com` hostname for resources within the `protectedApp` web application."
to 
"SPNEGO authentication is used for any requests received that contain the `example.com` hostname for resources within the `protectedApp` web application."

File: ref/feature/spnego/examples.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<2> The client's GSS credentials are not added to the user subject upon successful authentication.
+<3> The Kerberos configuration and keytab files to be used by the server are given specific locations within the server configuration directory instead of their respective default locations.
```
Comment (ManasiGandhi) [inline ref/feature/spnego/examples.adoc]: 'keytab files' should there be backticks for `keytab`?

Comment (Charlotte-Holt) [reply ref/feature/spnego/examples.adoc]: From what I was able to find, I don't think it does.

File: ref/feature/spnego/examples.adoc (hunk 2 above)
Comment (Charlotte-Holt) [reply ref/feature/spnego/examples.adoc]: Updated

File: ref/feature/spnego/examples.adoc (hunk 1 above)
Comment (Charlotte-Holt) [reply ref/feature/spnego/examples.adoc]: Yeah, I considered this too, but I don't think that I should use a slash between two words, so I've left as is.
